### PR TITLE
Enable Travis and AppVeyor CI

### PR DESCRIPTION
**Pull request by henkmollema**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  apt:
+    packages:
+    - gettext
+    - libcurl4-openssl-dev
+    - libicu-dev
+    - libssl-dev
+    - libunwind8
+    - zlib1g
+env:
+  - KOREBUILD_DNU_RESTORE_CORECLR=true KOREBUILD_TEST_DNXCORE=true
```
Comment (henkmollema) [inline .travis.yml]: How about these env vars, are they correct?

**Comment by henkmollema** [comment]
Green light, yay! :smile:

**Comment by anurse** [comment]
Soooo... I need you to rebase this on to release


... Kidding! Looks good! I'll merge it now! Sorry again for the run-around ;)

File: .travis.yml (hunk 1 above)
Comment (anurse) [inline .travis.yml]: I believe they are correct.

**Comment by henkmollema** [comment]
You almost had me :laughing:

**Comment by henkmollema** [comment]
@anurse can you add PlatformAbstractions to AppVeyor? It does not build yet: https://ci.appveyor.com/project/aspnetci/platformabstractions

**Comment by anurse** [comment]
I don't think I can, @Eilon can you?

**Comment by Eilon** [comment]
Enabled AppVeyor. The badge token is `m1ax1q33swwyaidm`.

**Pull request merged**